turbopy/core.py

```diff
@@ -611,15 +632,25 @@ def __init__(self, owner: Simulation, input_data: dict):
     def inspect_resource(self, resource: dict):
         """Save references to data from other PhysicsModules
 
-        If your subclass needs the data described by the key, now's their chance to
-        save a reference to the data
+        If your subclass needs the data described by the key, now's their
+        chance to save a reference to the data
+
+        Parameters
+        ----------
+        resource: dict
+            ????
```

Comment:
There is probably a better way to say this, but how about this:
```suggestion
            A dictionary containing references to data shared by other PhysicsModules.
```